apache/superset
# chore: bump superset to 0.18.6

### SUMMARY
Pull in the following PRs from `superset-ui`:
- https://github.com/apache-superset/superset-ui/pull/1371
- https://github.com/apache-superset/superset-ui/pull/1360
- https://github.com/apache-superset/superset-ui/pull/1370
- https://github.com/apache-superset/superset-ui/pull/1368

This PR also removes the direct dependency on `math-expression-evaluator`, as it's now a dependency in `superset-ui/core` that exposes the relevant functionality along with full test coverage.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. Create a formula annotation using both NVD3 and ECharts timeseries charts
2. Notice that formulas render like before

### ADDITIONAL INFORMATION
